lib/plugins/package/tests/zipService.js

```diff
@@ -216,4 +217,50 @@ describe('#zipService()', () => {
         .to.equal('node_modules/include-me/include-aswell');
     });
   });
+
+  it('should re-include files using ! glob pattern', () => {
+    const exclude = [
+      'node_modules/exclude-me/**',
+      'exclude-me/**',
+      '!exclude-me.js',
```

Comment:
Could you rename this to `include-me.js` so that it's more explicit what it should do?